app/src/components/fields/fieldErrorHint/fieldErrorHint.jsx

```diff
@@ -244,6 +244,10 @@ const messages = defineMessages({
     id: 'GenerateApiKeyModal.apiKeyNameUniqueHint',
     defaultMessage: 'API Key with the same name already exists',
   },
+  textMore128Hint: {
+    id: 'DeleteAccountFeedbackModal.textMore128Hint',
```

Comment:
Please don't use the validation values in message titles. Let it be just the part of the message itself.

Reply:
Fixed.